addons/web/static/src/js/views/kanban/kanban_record_quick_create.js

```diff
@@ -285,7 +285,7 @@ var RecordQuickCreate = Widget.extend({
         }
 
         // ignore clicks if target is inside the quick create
-        if (this.el.contains(ev.target) && this.el !== ev.target) {
+        if (this.el.contains(ev.target) || this.el === ev.target) {
```

Comment:
Hello @aab-odoo 

I would like your opinion on this fix when you have some time.

I'm not sure why the target needed to be different from the quick create element to be ignored.
This causes the quick create to close itself if the user clicks within the padding of the visual block.

So before:
"If you click on something inside AND NOT on the quick create element itself" -> leave the quick create open
After:
"If you click on something inside OR on the quick create element itself" -> leave the quick create open

It does not change a lot but I would like to be sure there is not a hidden reason why it worked that way.

Thanks!